rules/windows/sysmon/silenttrinity_stager_msbuild_activity.yml

```diff
@@ -0,0 +1,26 @@
+title: Silenttrinity Stager Msbuild Activity
+id: 50e54b8d-ad73-43f8-96a1-5191685b17a4
+description: Detects a possible remote connections to Silenttrinity c2
+references:
+    - https://www.blackhillsinfosec.com/my-first-joyride-with-silenttrinity/
+tags:
+    - T1127.001
+    - TA0002
+    - T1127
+    - attack.defense_evasion
+status: experimental
+author: Kiran kumar s, oscd.community
+date: 2020/10/11
+logsource:
+    category: sysmon
+    product: windows
+detection:
+    selection:
+        EventID: 3
```

Comment:
And delete this string.
I also ask you to periodically look at the existing rules. =)